### PR TITLE
Clean up main Dockerfile

### PR DESCRIPTION
**Pull request by alexshtin**
**What changed?**
Removed unused binaries from `Dockerfile`. 

**Why?**
Decrease docker image build speed.

**How did you test it?**
Build images locally and run them.

**Potential risks**
No risks.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -91,23 +89,24 @@ ENTRYPOINT ["tctl"]
...
 
-RUN apk add --update --no-cache jq
+RUN apk add --update --no-cache \
+    jq \
+    curl \
+    ca-certificates \
+    py-pip \
+    mysql-client \
+    postgresql-client \
+    && pip install cqlsh
```
Comment (alexshtin) [inline Dockerfile:99]: I try to run `cqlsh` and it fails with Python error:
```
bash-5.0# cqlsh
  File "/usr/bin/cqlsh", line 121
    except ImportError, e:
                      ^
SyntaxError: invalid syntax
```
I believe it is broken for a long time or had never worked. Any ideas how to fix it?

Comment (alexshtin) [reply Dockerfile:99]: Seems like wrong Python version.

Comment (wxing1292) [reply Dockerfile:99]: vaguely remember python 2 allows the above `,`

**Review by wxing1292 — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (alexshtin) [reply Dockerfile:100]: Ok, I fixed it.

**Pull request merged**